olm/olm-template/manifests/rhoas-operator.serviceregistryconnections.crd.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: serviceregistryconnections.rhoas.redhat.com
+  annotations:
+    service.binding/registryUrl: 'path={.status.registryUrl}'
+    ## Additional binding metadata required for Quarkus
+    service.binding/converter: 'path={.status.metadata.converter}'
```

Comment:
We need SSO url as well to obtain token. Additionally we missing credentials:
Values would be the same as in CLI. For example: 
https://identity.api.stage.openshift.com/auth/realms/rhoas

Reply:
Is the SSO URL the only thing missing and it is used to fetch credentials, or are the credentials needed to be embedded in the annotations?

Reply:
This will work exactly like Kafka SASL/OAUTH Bearer configuration. 
See this guide for details:
https://github.com/redhat-developer/app-services-guides/blob/main/quarkus/README.adoc#configuring-the-quarkus-example-application-to-connect-to-a-kafka-instance

This should make things clear. In short we need to replicate what we have for kafka (even names can be similar) + this time we need extra oauth URL to get token on app level (using credentials that will be also supplied)